packages/@o3r/core/schematics/ng-add/schema.ts

```diff
@@ -41,4 +41,10 @@ export interface NgAddSchematicsSchema extends SchematicOptionObject {
 
   /** Enable Apis manager */
   enableApisManager: boolean;
+
+  /** Initial git repository commit information. */
+  commit: boolean | { name: string; email: string };
```

Comment:
The comment is not so clear on the possible values. It is a bit strange to allow a boolean or an object so I would put more information on the commit

Reply:
it is actually the one from the official Angular `ng new` schematic.
(same for the fact to accept boolean and object, I just followed the exact same type)